src/main/scala/com/wavesplatform/transaction/smart/RealTransactionWrapper.scala

```diff
@@ -48,7 +49,8 @@ object RealTransactionWrapper {
       timestamp = o.timestamp,
       expiration = o.expiration,
       matcherFee = o.matcherFee,
-      signature = ByteVector(o.signature)
+      bodyBytes = ByteVector(o.bytes()), //TODO: Change to real bodyBytes
```

Comment:
Really, why not `bodyBytes`? It's called `toSign` now, but should probably be renamed